beancount/query/query_compile.py

```diff
@@ -90,6 +91,16 @@ def __init__(self, value):
     def __call__(self, _):
         return self.value
 
+class EvalAmount(EvalNode):
```

Comment:
Is this class used only in the unit tests? If so, why can't you simply use `EvalConstant` and pass an `amount.Amount` instance to its constructor?

Reply:
I can do and now do. Thanks.